proteinbenchmark/protein_system.py

```diff
@@ -178,11 +185,13 @@ def setup(self):
                 force_field_file=self.force_field_file,
                 water_model_file=self.water_model_file,
                 solvent_padding=solvent_padding,
+                setup_prefix = self.setup_prefix,
+                sim_platform = self.sim_platform,
             )
 
         # Minimize energy of solvated system with Cartesian restraints on
         # non-hydrogen solute atoms
-        if not exists_and_not_empty(self.minimized_pdb):
+        if self.sim_platform != 'gmx' and not exists_and_not_empty(self.minimized_pdb):
```

Comment:
Similarly to the parametrized system file, we should simplify this to something like `self.minimized_coords` that contains either a PDB or GRO file depending on the engine.